s3-source-connector/src/test/java/io/aiven/kafka/connect/s3/source/utils/FileReaderTest.java

```diff
@@ -141,14 +255,30 @@ private S3ObjectSummary createObjectSummary(final long sizeOfObject, final Strin
         return summary;
     }
 
-    private void setBasicProperties() {
-        properties.put(S3SourceConfig.INPUT_FORMAT_KEY, InputFormat.BYTES.getValue());
-        properties.put("name", "test_source_connector");
-        properties.put("key.converter", "org.apache.kafka.connect.converters.ByteArrayConverter");
-        properties.put("value.converter", "org.apache.kafka.connect.converters.ByteArrayConverter");
-        properties.put("tasks.max", "1");
-        properties.put("connector.class", AivenKafkaConnectS3SourceConnector.class.getName());
-        properties.put(TARGET_TOPIC_PARTITIONS, "0,1");
-        properties.put(TARGET_TOPICS, "testtopic");
+    private Iterator<S3ObjectSummary> getS3ObjectSummaryIterator(final String objectKey) {
+        final S3ObjectSummary objectSummary = createObjectSummary(1, objectKey);
+        final ListObjectsV2Result listObjectsV2Result = createListObjectsV2Result(
+                Collections.singletonList(objectSummary), null);
+        when(s3Client.listObjectsV2(any(ListObjectsV2Request.class))).thenReturn(listObjectsV2Result);
+        when(offsetManager.getOffsets()).thenReturn(new HashMap<>());
+
+        return fileReader.fetchObjectSummaries(s3Client);
+    }
+
+    public void initializeWithTaskConfigs(final int maxTasks, final int taskId) {
+        final S3SourceConfig s3SourceConfig = mock(S3SourceConfig.class);
+        final Map<String, Object> configMap = getObjectMap(maxTasks, taskId);
+
+        when(s3SourceConfig.originals()).thenReturn(configMap);
+        offsetManager = mock(OffsetManager.class);
+        fileReader = new FileReader(s3SourceConfig, TEST_BUCKET, Collections.emptySet());
+        s3Client = mock(AmazonS3.class);
```

Comment:
Same here

Reply:
And for both annotations are not needed if they are not automatically created

Reply:
removed those mock annotations. indeed.